client-js/main/client/actor-request-factory.js

```diff
@@ -647,6 +655,53 @@ class QueryBuilder extends AbstractTargetBuilder {
     return this;
   }
 
+  /**
+   * Requests the query results to be ordered by the given `column` in the descending direction.
+   *
+   * Weather or not the results will be sorted in the given order depends on the implementation
```

Comment:
`Whether the results will be sorted in the requested order depends on the implementation of server-side communication.`